packages/cactus-plugin-htlc-eth-besu-erc20/README.md

```diff
@@ -0,0 +1,44 @@
+# `@hyperledger/cactus-plugin-htlc-eth-besu-erc20`
+
+Allows Cactus nodes to interact with HTLC ETH SWAP contracts with ERC-20 Tokens
+
+## Summary
+
+  - [Getting Started](#getting-started)
+  - [Runing the tests](#running-the-tests)
+  - [Deployment](#deployment)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [License](#license)
+
+## Getting Started
+
+### Prerequisites
+
+In a Cactus root directory you need execute:
+```
+   npm run run-ci
+```
+### Installing
+
+In this directory you can execute:
+```
+    npm run tsc
```

Comment:
@jordigiam Could you confirm if this really works while standing in the package's directory instead of the project root?

Reply:
Fixed